enterprise/models.py

```diff
@@ -2235,14 +2237,27 @@ class Meta:
         'placeholder {course_name} will be replaced with the name of the course or program that was enrolled in.'
     )
 
+    DEFAULT_TEMPLATE_TYPE = 'DE'
+    BULK_ENROLL_TEMPLATE_TYPE = 'BE'
+    template_type_choices = [
+        (DEFAULT_TEMPLATE_TYPE, 'Default Enrollment Template'),
+        (BULK_ENROLL_TEMPLATE_TYPE, 'Bulk Enrollment Template'),
+    ]
+
     plaintext_template = models.TextField(blank=True, help_text=BODY_HELP_TEXT)
     html_template = models.TextField(blank=True, help_text=BODY_HELP_TEXT)
     subject_line = models.CharField(max_length=100, blank=True, help_text=SUBJECT_HELP_TEXT)
-    enterprise_customer = models.OneToOneField(
+    enterprise_customer = models.ForeignKey(
         EnterpriseCustomer,
-        related_name="enterprise_enrollment_template",
+        related_name="enterprise_enrollment_templates",
         on_delete=models.deletion.CASCADE
     )
+    template_type = models.CharField(
+        max_length=2,
```

Comment:
yeah ok with the upcoming change we will want a larger value for sure, will do 255 in case we add other values in the future.

Reply:
yeah I don't think it necessarily has to be 255. It can be whatever. I appreciate the consideration of the actual data length and I don't see us writing a paragraph for a label (I will veto that anyways), so I think you can stick with your gut to limit it, I just wouldn't go with 2.

Reply:
>  I don't see us writing a paragraph for a label

meaning help_text or?